### PR TITLE
Dockerswarm

### PR DESCRIPTION
**Pull request by Notaduck**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+NUM_OF_MANAGERS: 2
```
Comment (einarklarlund) [inline config.yml:1]: This file should be removed

Comment (Notaduck) [reply config.yml:1]: I agree

File: nginx/nginx.conf [hunk 2]
```diff
@@ -10,10 +10,12 @@ http {
   sendfile on;
 
   upstream web_up {
+    least_conn;
     server web:8000;
   }
 
   upstream api_up {
+    least_conn;
```
Comment (einarklarlund) [inline nginx/nginx.conf:18]: need to include in the report that we are using nginx for load balancing now

File: report/out/main.tex [hunk 3]
```diff
@@ -0,0 +1,47 @@
+\hypertarget{cicd}{%
```
Comment (einarklarlund) [inline report/out/main.tex]: This file should be removed

**Pull request merged**